ksql-engine/src/test/java/io/confluent/ksql/KsqlContextTest.java

```diff
@@ -71,15 +77,25 @@ public void shouldRunSimpleStatements() throws Exception {
     KafkaStreams queryStreams = mock(KafkaStreams.class);
     queryStreams.start();
     expectLastCall();
+
+
+    OutputNode outputNode = mock(OutputNode.class);
+    expect(outputNode.accept(anyObject(), anyObject())).andReturn(null);
```

Comment:
Could we be more specific about what we expect the arguments of this call to be? For instance, that we expect the visitor to be of type `PlanSourceExtractorVisitor`?